internal/pkg/exec/docker.go

```diff
@@ -230,6 +246,21 @@ func (c DockerCommand) IsEcrCredentialHelperEnabled(uri string) bool {
 	return false
 }
 
+// ValidatePlatform checks if the entered string is a Docker-buildable platform.
+func ValidatePlatform(platform string) error {
+	if platform == "" {
+		return nil
+	}
+	osArch := strings.Split(platform, "/")
+	if len(osArch) < 2 {
+		return fmt.Errorf("platform %s is invalid; must be of format 'os/arch'", platform)
+	}
+	if osArch[0] != LinuxOS || osArch[1] != Amd64Arch {
+		return fmt.Errorf("platform %s is invalid; valid platforms are: %s", platform, ValidPlatforms)
```

Comment:
We should take advantage of this package https://github.com/dustin/go-humanize here!

For example,
```suggestion
		return fmt.Errorf("platform %s is invalid; %s: %s", 
		english.PluralWord(len(ValidPlatforms), "the valid platform is", "valid platforms are"),
		english.WorkSeries(ValidPlatforms, "and"))
```